src/pluto.py

```diff
@@ -500,14 +500,19 @@ def generate_graph(v1, v2):
                 "pfirm15_flag",
             ]
             x = [(v1[i] / v2[i] - 1) * 100 for i in y]
+            real_v1 = [v1[i] for i in y]
+            real_v2 = [v2[i] for i in y]
             hovertemplate = "<b>%{x} %{text}</b>"
+            #add tooltips with order <percent change, value from v1, value from v2>
+            tooltip= zip([round(a,2) for a in x],[round(b,2) for b in real_v1],[round(c,2) for c in real_v2])
```

Comment:
Are the three numbers percent change, new value, old value? I think we need some way to communicate that to the user. 
The % sign should be next to the first value, not at the end I think